### PR TITLE
move tests from legacy

### PR DESCRIPTION
**Pull request by mkeskells**
Tests use new model output

### CODE REVIEW

**Comment by dwijnand** [comment]
If you give me repo write access (I think it's that one that I need) I could rebase these for you.

File: testProjects/unitTestProject/src/main/scala/scalaclean/test/references/allOutgoingReferences/allOutgoingReferences.scala.expected [hunk 1]
```diff
@@ -34,11 +34,11 @@ object XX {
   println(Special.myVar2)
 
   Special()
-  val x: Any = 1/* allOutgoingReferences(G:scalaclean/test/references/allOutgoingReferences/XX.x.) - <root>/scala :: <root>/scala/Any# */
+  val x: Any = 1/* allOutgoingReferences(<root>/scalaclean/test/references/allOutgoingReferences/XX/x .) - <root>/scala :: <root>/scala/Any# */
```
Comment (dwijnand) [inline testProjects/unitTestProject/src/main/scala/scalaclean/test/references/allOutgoingReferences/allOutgoingReferences.scala.expected:37]: I think the trailing space is from nsc, where it's some kind of indicator of "local name", I think?

Comment (mkeskells) [reply testProjects/unitTestProject/src/main/scala/scalaclean/test/references/allOutgoingReferences/allOutgoingReferences.scala.expected:37]: Yes - its a filed. We should probably handle that specially, like we do for methods

Comment (dwijnand) [reply testProjects/unitTestProject/src/main/scala/scalaclean/test/references/allOutgoingReferences/allOutgoingReferences.scala.expected:37]: filed?

Comment (rorygraves) [reply testProjects/unitTestProject/src/main/scala/scalaclean/test/references/allOutgoingReferences/allOutgoingReferences.scala.expected:37]: field

**Pull request merged**